tests/tasks/create_test_interfaces.yml

```
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Install dnsmasq
+  package:
+    name: dnsmasq
+    state: present
+
+
+- name: Create test interfaces
+  shell: |
+    # Manage all veth devices starting with test*
+    echo 'ENV{ID_NET_DRIVER}=="veth", ENV{INTERFACE}=="test*", ENV{NM_UNMANAGED}="0"' >/etc/udev/rules.d/88-veth.rules
+    udevadm control --reload-rules
+    udevadm control --reload-rules
+
+    # Create a namespace for devices
+    ip netns add testsetup
```

```suggestion
    ip netns add lsrdhcp
```

To avoid conflicts with other tests.

---

done